### DISCUSSION TITLE
3D subdivisions

### DISCUSSION THREAD

**Discussion by joex92** [GriefPrevention/GriefPrevention#1570] [Ideas]
What about adding a feature to the subdivision command to be able to create 3D subdivisions?
that way, for example, we could have a building with different levels and share levels with other players...
or underground mining would be kept protected while they have access to a chest room...

**Comment by Jikoo** [comment]
Duplicate of #269

**Reply by joex92** [reply]
oh ok... is it going to be implemented? is it planned?

**Reply by Jikoo** [reply]
Since this is a duplicate comment on your duplicate issue, I will be replying to you over there.
